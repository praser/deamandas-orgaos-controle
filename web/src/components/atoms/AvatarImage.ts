import styled from 'styled-components';

interface Props {
  src: string;
  alt: string;
}

const AvatarImage = styled.img<Props>`
  width: 100%;
  max-width: 300px;
  height: auto;
`;

export default AvatarImage;
