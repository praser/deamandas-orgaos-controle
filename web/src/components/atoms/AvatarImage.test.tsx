import React from 'react';
import { render, screen } from '@testing-library/react';
import AvatarImage from './AvatarImage';

describe('Avatar image atomic component', () => {
  const alt = 'avatar';
  const src = 'some-image.png';
  let avatarImage: HTMLElement;

  beforeEach(() => {
    render(<AvatarImage src={src} alt={alt} />);
    avatarImage = screen.getByAltText(alt);
  });

  it('is expected to math the snapshot', () => {
    expect(avatarImage).toMatchSnapshot();
  });
});
