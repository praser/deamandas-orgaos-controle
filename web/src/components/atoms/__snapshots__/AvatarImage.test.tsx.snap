// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar image atomic component is expected to math the snapshot 1`] = `
<img
  alt="avatar"
  class="sc-bdvvaa plTdr"
  src="some-image.png"
/>
`;
